### PR TITLE
Add James Azam's details

### PR DESCRIPTION
**Pull request by jamesmbaazam**
The title above is self-explanatory.

### CODE REVIEW

**Review by Bisaloo — COMMENTED** [review]
Thanks for getting the ball rolling!

Not sure what you prefer or if that even fits in your case but just in case: `person()` has an argument for `middle` name in the 3rd position.

Let's also try to standardize a couple of things to avoid that each contributor makes formatting changes:

- should we use the short (`c(ORCID = "0000-0001-5782-7330")`) or long (c(ORCID = "https://orcid.org/0000-0001-5782-7330")`) format for the ORCID comment? The former is more compact but the latter makes the link easily clickable.
- can you keep the [tidyverse style](https://github.com/tidyverse/ggplot2/blob/d7f22413efea3dd2a7c9effff05d4b2aa2c2d300/DESCRIPTION#L4-L22) please? I.e., only one line for each `person` expected for the comment.

**Comment by jamesmbaazam** [comment]
> `person()` has an argument for `middle` name in the 3rd position.

I remember `lintr` complains when you use "middle". I may be wrong.

**Comment by Bisaloo** [comment]
> I remember lintr complains when you use "middle". I may be wrong.

I just tried it locally and it seems to work. Maybe it was a temporary bug. I'd suggest to try again.

**Comment by Bisaloo** [comment]
Oh, good catch, thanks for letting me know!

File: DESCRIPTION [hunk 1]
```diff
@@ -1,9 +1,12 @@
...
 Version: 0.0.1
-Authors@R: 
-    c(person("Thibaut", "Jombart", , "thibaut@data.org", role = c("aut", "cre")),
-      person("Avinash", "Laddha", , "avinash@data.org", role = c("aut")))
+Authors@R: c(
+    person(given = "Thibaut", family = "Jombart", , email = "thibaut@data.org", role = c("aut", "cre")),
+    person(given = "Avinash", family = "Laddha", , email = "avinash@data.org", role = c("aut")),
+    person(given = "James M.", family = "Azam", , email = "james.azam@lshtm.ac.uk", role = c("aut"),
+           comment = c(ORCID = "https://orcid.org/0000-0001-5782-7330"))
+)
```
Comment (Bisaloo) [inline DESCRIPTION:9]:
```suggestion
  )
```

The current format fails R CMD check.

**Comment by Bisaloo** [comment]
Thanks!

**Pull request merged**